packages/manager/apps/container/src/core/container/ContainerProvider.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useReket } from '@ovh-ux/ovh-reket';
+
+import ContainerContext from './context';
+
+export const ContainerProvider = ({ children }) => {
+  const reketInstance = useReket();
+  const preferenceKey = 'NAV_RESHUFFLE_BETA_ACCESS';
+  const [isLoading, setIsLoading] = useState(true);
+  const [isBeta, setIsBeta] = useState(false);
+  const [askBeta, setAskBeta] = useState(false);
+
+  let containerContext = useContext(ContainerContext);
+
+  const updateBetaChoice = async (accept = false) => {
+    return reketInstance
+      .put(`/me/preferences/manager/${preferenceKey}`, {
+        key: preferenceKey,
+        value: accept ? 'true' : 'false',
+      })
+      .then((result) => {
+        if (accept === false) {
```

Comment:
```suggestion
        if (!accept) {
```
would be sufficient no ?